lightly_studio/tests/helpers_resolvers.py

```diff
@@ -350,6 +352,30 @@ def create_samples_with_embeddings(
     return result
 
 
+def create_caption(
+    session: Session,
+    dataset_id: UUID,
+    parent_sample_id: UUID,
+    text: str,
```

Comment:
Please set a default for tests that don't need the text value (e.g. "test_caption")